perfume-api/src/main/java/io/perfume/api/sample/application/port/SampleRepository.java

```diff
@@ -0,0 +1,8 @@
+package io.perfume.api.sample.application.port;
+
+import io.perfume.api.sample.domain.Sample;
+
+public interface SampleRepository {
+
+    Sample save(Sample entity);
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 4, expected level should be 2.